project/Build.scala

```diff
@@ -14,6 +14,10 @@ object Scrooge extends Build {
   val suffix = if (branch == "master") "" else "-SNAPSHOT"
 
   val libVersion = "3.18.1" + suffix
+
+  // For develop users: if you are using new features in util and finagle dependencies
```

Comment:
Also, it might be a good idea to recommend publishing Util and Finagle locally for _anyone_ working on the develop branch—users may not have any idea whether Scrooge is using features that have been introduced since the last release, and if they guess wrong they may end up with weird errors (possibly at runtime).